src/kite.js

```diff
@@ -231,8 +215,32 @@ const Kite = {
     );
 
     this.disposables.push(
-      vscode.commands.registerTextEditorCommand("kite.related-files", (textEditor) => {
-        opn(`http://localhost:46624/codenav/related/${encodeURIComponent(textEditor.document.fileName)}`);
+      vscode.commands.registerTextEditorCommand("kite.related-code-from-file", (textEditor) => {
+        KiteAPI
+          .requestRelatedCode(textEditor.document.fileName, null, null, "vscode")
+          .catch(NotificationsManager.getRelatedCodeErrHandler(textEditor.document.fileName, 0))
+      })
+    );
+
+    this.disposables.push(
+      vscode.commands.registerTextEditorCommand("kite.related-code-from-line", (textEditor) => {
+        const zeroBasedLineNo = textEditor.selection.active.line
+        const oneBasedLineNo = zeroBasedLineNo+1
```

Comment:
If across the board, editors refer to lines as zero indexed, I think it might be cleaner for the plugin if Kited internally increments the value instead. If not, then this implementation works.

Reply:
some quick searching: Atom also uses 0-based, but in vim `line(".")` returns a 1-based line number. I suppose I'm inclined to keep it as it is